packages/help-center/src/hooks/use-is-wapuu-enabled.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-restricted-imports */
+import config from '@automattic/calypso-config';
+import { useSelector } from 'calypso/state';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { useSupportAvailability } from '../data/use-support-availability';
+
+/**
+ * Helper hook to determine if Wapuu should be enabled.
+ * @returns {boolean} Whether Wapuu should be enabled or not.
+ * @example
+ * const isWapuuEnabled = useIsWapuuEnabled();
+ * if ( isWapuuEnabled ) {
+ *     // Do something
+ * }
+ */
+export const useIsWapuuEnabled = () => {
+	const { data: supportAvailability, isLoading } = useSupportAvailability( 'OTHER' );
+	const isFreeUser = ! supportAvailability?.is_user_eligible_for_tickets;
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const isSiteJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+	const isWapuuConfigEnabled = config.isEnabled( 'wapuu' );
```

Comment:
For this to work in the editor (outside Calypso), you'd need to add this config [here](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/help-center/src/config.js#L0-L1).